src/ol_infrastructure/applications/learn_ai/Pulumi.applications.learn_ai.QA.yaml

```diff
@@ -23,9 +23,12 @@ config:
     NODE_ENV: "development"
     POSTHOG_API_HOST: "https://app.posthog.com"
     POSTHOG_ENABLED: "true"
+    POSTHOG_ENABLE_SESSION_RECORDING: "true"
     POSTHOG_PROJECT_ID: "86325"
     POSTHOG_TIMEOUT_MS: "1000"
-    POSTHOG_ENABLE_SESSION_RECORDING: "true"
+    SENTRY_LOG_LEVEL: "ERROR"
+    SENTRY_PROFILES_SAMPLE_RATE: 0
+    SENTRY_TRACES_SAMPLE_RATE: 0
```

Comment:
We probably want to turn this up a bit, maybe 0.1?

Reply:
Oh sure, I didn't even know what they are for, I just copied Matt's defaults from his PR.